dll/win32/advapi32/advapi32.spec

```diff
@@ -220,17 +220,17 @@
 @ stdcall EnumServicesStatusExW(long long long long ptr long ptr ptr ptr wstr)
 @ stdcall EnumServicesStatusW(long long long ptr long ptr ptr ptr)
 @ stdcall -version=0x502 EnumerateTraceGuids(ptr long ptr) ntdll.EtwEnumerateTraceGuids
-@ stdcall -stub -version=0x600+ EnumerateTraceGuids(ptr long ptr)
+@ stdcall -stub -version=0x600+ EnumerateTraceGuids(ptr long ptr) # EtwEnumerateTraceGuids
```

Comment:
Is this commented out on purpose? Same question further down...

Reply:
No, on NT6+ it's not a forwarder, it's implemented in advapi32. The comment is just a hint regarding the different name. It should eventually be implemented in the eetwtrace lib, I was just too lazy to add stubs there.